### PR TITLE
Update migration guide - Closes #1155

### LINKED ISSUE #1155: Migration guide updates for Mainnet
- Bump lisk-migrator version to `1.0.0`

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1155

### CODE REVIEW

**Review by shuse2 — DISMISSED** [review]

**Review by nazarhussain — DISMISSED** [review]

File: modules/ROOT/pages/management/migration.adoc [hunk 1]
```diff
@@ -219,7 +219,11 @@ mv $TEMP_FILE ~/lisk-core/config/testnet/custom-config.json
 
 === Check the announced snapshot height
 
-Check the announced snapshot height on {url_lisk_blog_migration}[^].
+The announced snapshot heights
+
+//TODO: Add link to mainnet migration blog post
+* For Mainnet: `16026074`
```
Comment (Nazgolze) [inline modules/ROOT/pages/management/migration.adoc]: Since this height is just for testing, do we really want this here?

Comment (Tschakki) [reply modules/ROOT/pages/management/migration.adoc]: Hmm good point. Maybe we can pin it in the testnet channel, together with a link to the guide. Will remove it here

File: modules/ROOT/pages/setup/binary.adoc [hunk 2]
```diff
@@ -42,17 +42,31 @@ Download Lisk Core:
...
 [source,bash]
 ----
-curl -o lisk-core.tar.gz https://downloads.lisk.io/lisk/testnet/3.0.0-rc.4/lisk-core-v3.0.0-rc.4-linux-x64.tar.gz
+curl -o lisk-core.tar.gz https://downloads.lisk.io/lisk/mainnet/3.0.0-rc.4/lisk-core-v3.0.0-rc.4-linux-x64.tar.gz
+----
+
+*macOS*
+[source,bash]
+----
+curl -o lisk-core.tar.gz https://downloads.lisk.io/lisk/mainnet/3.0.0-rc.4/lisk-core-v3.0.0-rc.4-darwin-x64.tar.gz
```
Comment (Nazgolze) [inline modules/ROOT/pages/setup/binary.adoc]: This doesn't exist

Comment (Nazgolze) [reply modules/ROOT/pages/setup/binary.adoc]: 3.0.0 will be the first 3.x version https://downloads.lisk.io/lisk/mainnet, when the time comes :+1:

Comment (Tschakki) [reply modules/ROOT/pages/setup/binary.adoc]: What does this mean? are we using testnet binary version for the mainnet migration test?

**Pull request merged**